docs/modules/trino/pages/usage-guide/operations/graceful-shutdown.adoc

```diff
@@ -1,32 +1,65 @@
 = Graceful shutdown
 
-== How it works
+You can configure the graceful shutdown as described in xref:concepts:operations/graceful_shutdown.adoc[].
+
+== Coordinators
+
+As a default, coordinators have `15 minutes` to terminate gracefully.
+
+The coordinator process will always run as PID `1` and will get a `SIGTERM` once Kubernetes wants to terminate the Pod.
+It will initiate a graceful shutdown.
+After the graceful shutdown timeout is passed and the process still didn't exit, Kubernetes will issue an `SIGKILL` to force-kill the process.
+
+When a coordinator gets restarted all running queries will fail and will not be recovered.
+As of Trino version `429` this can not be prevented (e.g. by using multiple coordinators).
```

Comment:
```suggestion
Starting with Trino version `429` this behavior can not be prevented (e.g. by using multiple coordinators).
```

Reply:
It actually was never supported. We just put in the version 428/429 here to be still correct, even when e.g. version 450 adds support for that.